modules/oe_content_entity/tests/src/FunctionalJavascript/StakeholderUiTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_entity\FunctionalJavascript;
+
+use Drupal\media\Entity\Media;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Test Stakeholder bundle.
+ */
+class StakeholderUiTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'oe_content',
+    'oe_content_entity',
+    'oe_content_entity_contact',
+    'oe_content_entity_organisation',
+    'oe_content_entity_venue',
+  ];
+
+  /**
+   * Tests Stakeholder creation.
+   */
+  public function testStakeholderBundleUi(): void {
+    // Create dummy media entity image.
+    $file_data = file_get_contents('https://upload.wikimedia.org/wikipedia/commons/4/45/Eopsaltria_australis_-_Mogo_Campground.jpg');
```

Comment:
I dont like this hard coded external url, please add a fixtures folder and add test file there